HeavyIonsAnalysis/JetAnalysis/interface/HiInclusiveJetAnalyzer.h

```diff
@@ -74,6 +74,11 @@ class HiInclusiveJetAnalyzer : public edm::EDAnalyzer {
 
 private:
 
+  // for reWTA reclustering-----------------------
+  bool doWTARecluster_ = false;
+  fastjet::JetDefinition WTAjtDef = fastjet::JetDefinition(fastjet::JetAlgorithm::antikt_algorithm, 1, fastjet::WTA_pt_scheme);
```

Comment:
Is the intention to recluster *all* constituents with WTA and get the jet axis?  If so, I would increase the 1 larger.  Imagine we run this for a wide jet and the WTA reclustering gives you two jets.

Reply:
I think that this has been planned to be used with R=0.4 jets, in which case 1 should probably be large enough radius. For wider jet radius parameters this might have to be increased.

Reply:
Why don't we just put 2.0 or something and don't have to worry in the future?

Reply:
That can be done This should not change the functionality for smaller jet radii?

Reply:
Yes, setting it arbitrary large should be fine since we collect only the candidates for the given jet.